src/Timer/Timer.php

```diff
@@ -59,11 +52,11 @@ public function isPeriodic()
 
     public function isActive()
     {
-        return $this->loop->isTimerActive($this);
+        return l\loop()->isTimerActive($this);
```

Comment:
IMHO it makes sense to no longer expose the loop as part of the timer interface (i.e. drop the `getLoop()` method).

Though arguably, I'm also pro exposing the one loop via global state.

However, I consider this very change a design flaw. Unless we move this logic to the timer itself, it *actually does* have a dependency on the loop. Accessing global state here means we're using a leaky abstraction and hiding the dependencies this particular class has. Also, this does make testing harder (no built-in mechanism to mock the loop).